server/views/partials/layout.njk

```diff
@@ -100,6 +100,11 @@
   <script src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
   <script src="/assets/print.js"></script>
+  <script src="/assets/applicationinsights-web.min.js"></script>
```

Comment:
This line loads the App Insights library code from our assets static resources path (which in turn comes from the node_modules folder)